pycue/opencue/util.py

```diff
@@ -40,27 +41,27 @@ def grpcExceptionParser(grpcFunc):
     """Decorator to wrap functions making GRPC calls.
     Attempts to throw the appropriate exception based on grpc status code."""
     def _decorator(*args, **kwargs):
-        try:
-            return grpcFunc(*args, **kwargs)
-        except grpc.RpcError as exc:
-            code = exc.code()
-            details = exc.details() or "No details found. Check server logs."
-            if code == grpc.StatusCode.NOT_FOUND:
-                future.utils.raise_with_traceback(opencue.exception.EntityNotFoundException(
-                    "Object does not exist. {}".format(details)))
-            elif code == grpc.StatusCode.ALREADY_EXISTS:
-                future.utils.raise_with_traceback(opencue.exception.EntityAlreadyExistsException(
-                    "Object already exists. {}".format(details)))
-            elif code == grpc.StatusCode.DEADLINE_EXCEEDED:
-                future.utils.raise_with_traceback(opencue.exception.DeadlineExceededException(
-                    "Request deadline exceeded. {}".format(details)))
-            elif code == grpc.StatusCode.INTERNAL:
-                future.utils.raise_with_traceback(opencue.exception.CueInternalErrorException(
-                    "Server caught an internal exception. {}".format(details)))
-            else:
-                future.utils.raise_with_traceback(opencue.exception.CueException(
-                    "Encountered a server error. {code} : {details}".format(
-                        code=code, details=details)))
+        triesRemaining = opencue.exception.getRetryCount()
```

Comment:
Should this be `getRetryCount() + 1`?

I'm just thinking, if I set `exception_retries` to zero I would expect the call to be made once still.

Reply:
Good call. PTAL.